spec/PeriodicBackgroundSync-index.bs

```diff
@@ -145,6 +148,30 @@ Fetch requests within the [=periodicsync event=] while [=periodicsync event/in t
 ## History Leaking ## {#history-leaking}
 Fetch requests within the [=periodicsync event=] while [=periodicsync event/in the background=] may reveal something about the client's navigation history to middleboxes on networks different from the one used to create the [=periodic sync registration=]. For instance, the client might visit site https://example.com, which registers a [=periodicsync event=], but based on the implementation, might not fire until after the user has navigated away from the page and changed networks. Middleboxes on the new network may see the fetch requests that the [=periodicsync event=] makes. The fetch requests are HTTPS so the request contents will not be leaked but the destination of the fetch request and domain may be (via DNS lookups and IP address of the request). To prevent this leakage of browsing history, the user agent MAY choose to only fire [=periodicsync event|periodicsync events=] on the network the [=periodic sync registration=] was made on, with the understanding that it will reduce usability by not allowing synchronization opportunistically.
 
+Security Considerations {#security}
+====================================
+Since Periodic Background Sync is a [=service worker=] based API, [[service-workers#security-considerations]] apply.
+
+In addition, if there is a disparity between the intervals at which the web app can synchronize with its servers and those at which the user visits the site, the [=origin=] gets more power than the user is aware of or intends. This is because the [=origin=] is able to execute code more frequently than the user intends, from within the [=periodicsync event=] while [=periodicsync event/in the background=].
```

Comment:
Try to be more precise than just "more power". What attacks get opened up? Most of the mitigations should be required or recommended elsewhere in the specification, and then this section should explain how those mitigations help prevent the possible attacks.

Reply:
Also: it's not just any disparity, it's if the origin synchronizes more often (thanks to PBS) than the user visits the site.